### PR TITLE
Avoid outputting color unless terminal able to render it

### PR DESCRIPTION
**Pull request by knz**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -28,6 +28,7 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/cockroachdb/cockroach/pkg/util/color"
```
Comment (tamird) [inline main.go]: This is a pretty hefty dependency. Is it worth avoiding?

**Comment by knz** [comment]
What's hefty about it? There are just two files in the package.
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by tamird** [comment]
You're introducing a dependency on the entire github.com/cockroachdb/cockroach repository. `go get` is not able to retrieve just a single package; the entire repository is fetched.

**Comment by knz** [comment]
Oh thanks for the explanation. Fixed. PTAL.

**Comment by knz** [comment]
thanks!

**Comment by benesch** [comment]
Wait this getColors() function seems like a silly layer of indirection. You’re swapping named constants for positional return values. Mind if I refactor to use os.Stdout(ttycolor.BLAH)?

**Comment by knz** [comment]
Be my guest
-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Pull request merged**